recipes/toast/LICENSE

```diff
@@ -0,0 +1,28 @@
+
+Time Ordered Astrophysics Scalable Tools (TOAST)
```

Comment:
Looks like the License file is part of the tarball, so please remove this "LICENCE" file from the recipe dir.

Reply:
Ok, @npavlovikj , file removed.  Thanks for your review.